### DISCUSSION TITLE
New Component - Validation code

### DISCUSSION THREAD

**Discussion by turolopezsanabria** [SUI-Components/sui-components#1574] [Ideas and new features]
Here we will add all the features needed for this new component:
- [ ] Focus in the field that the user needs to fill the following number.
- [ ] When the user enters the number, automatically the focus goes to the next field.
- [ ] By default, it will be 6 numbers to fill although you can decide how many fields do you want at the begining.
- [ ] In mobile devices, when the SMS arrives, the field must be proposed to autofilled with the SMS number recieved.
- [ ] We can add an erase button to delete the selection.
- [ ] Before verify the code, we can add a button to resend.
- [ ] ¿Proceed verification automatically when the user insert the last number?

React Verification code input example: https://github.com/suweya/react-verification-code-input 

**Specs**

https://www.figma.com/file/KEuHkTBkjjb1VX9D2dd1fn/?node-id=0%3A1

**Comment by turolopezsanabria** [comment]
To the last point, I'm not sure
Any **cons** doing so?

**Comment by nuria-isart** [comment]
Specs of the component: https://www.figma.com/file/KEuHkTBkjjb1VX9D2dd1fn/New-Component-Verification-Code?node-id=522%3A478

**Comment by turolopezsanabria** [comment]
Hey there, small detail
in our component, I always get the suggestion to use a saved password (something that we don't need at all for this case)

![savedpassword](https://user-images.githubusercontent.com/23620759/126649801-98fdcf8d-3564-48dd-89a2-e7d6d79769de.png)

Can we disable it?

On an external example, this is from Figma today, no suggestion to use a saved password.

https://user-images.githubusercontent.com/23620759/126649876-15abbd78-f8ce-4c1a-a5d1-8e9e1f56751a.mov

**Comment by nistadev** [comment]
I would also suggest to add a "delete from keyboard feature", where the user press `Del` key, removing the last introduced value, focusing in the previous input. Here's an example from suweya's [demo](https://suweya.github.io/react-verification-code-input/).

![0ed183eae046fc06c60b3a355cd5cb20](https://user-images.githubusercontent.com/22194171/126997544-7dca59bf-bc70-4880-a905-b124fb8122ca.gif)

**Reply by Alverd04** [reply]
I totally agree!

**Reply by andresin87** [reply]
@Alverd04 is attending this request
